lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
Following this reasoning, all custom elements should be removed altogether. I see no reason why.